client/src/App.jsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import styled from 'styled-components';
 import Thumbnails from './components/Photos.jsx';
-import Modal from './components/Modal.jsx';
+import ZoomPopover from './components/ZoomPopover.jsx';
 
 const PhotosWrapper = styled.div`
```

Comment:
I like that you are keeping your styled components in the file it's being used in. I have mine separated out but I think this way keeps it better organized and less confusing.

Reply:
I initially had the same idea as you, then I tried to keep only some more global styles in a separate file but I ended up with only two styled components in that file so I just got rid of it entirely.